### DISCUSSION TITLE
Customize component - customize toolbar

### DISCUSSION THREAD

**Discussion by paisCFL** [n1crack/vuefinder#41] [Q&A]
Hi Yusuf,

I'm trying to understand if your component suits what I need, but I'm not sure if there is a way to customize the component because I didn't find documentation that clarifies this, such as adding a button to the toolbar. Is it possible for you to help me know this?

Thank you and congratulations for what you have here.

**Comment by n1crack** [comment]
Hi, thank you considering the library. I hope you can achieve what you want. 

Adding a button to the toolbar is not possible (yet). But instead you can enable a button in the status bar. And you have full control of it 
 and the usage is very simple,

What exactly do you want to achieve? Ultimately, if it is generally useful, it is likely to be added.


![image](https://github.com/n1crack/vuefinder/assets/712404/df3d2476-2bfa-4b40-b02d-103c8fc3867b)


![image](https://github.com/n1crack/vuefinder/assets/712404/a2c4f03e-7570-4d1a-9f73-0ee94d54d1e7)
